src/docs/user/references/ScriptLanguageSupport.adoc

```diff
@@ -396,6 +396,50 @@ Or another script binding (for example, USERSPACE):
 my $USERSPACE= $ENV{"USERSPACE"};
 ----
 
+=== PHP
+
+The PHP script engine features the following behaviors:
+
+* Script bindings are translated as environment variables.
+* Workflow variables are un-modifiable except by using a post-script written in another language such as the `update_variables_from_file` script (see <<_bash_and_shell,Bash script language>> for further explanations).
+* PHP script engine is meant to return HTML contents and makes use of link:../user/ProActiveUserGuide.html#_assigning_metadata_to_task_result[result metadata] through the *content.type*="text/html" link:../user/ProActiveUserGuide.html#_generic_information[generic information]. Using the script engine without defining this generic information in the Task is not recommended.
+* The _result_ of the script engine execution is a byte array containing the html content produced by the `php` command. In case the `php` command returns a non zero value, the result of the script engine execution will be the command exit code and the associated workflow task will be in error.
```

Comment:
We need to choose either use 'HTML' or 'html' in the whole document.

Reply:
ok for HTML